src/integrationtest/python/steps/step_application.py

```diff
@@ -6,7 +6,7 @@
 from jobs import Jobs
 from settings import Settings
 from behave import use_step_matcher, when, then
-from src.integrationtest.python.steps.step_helpers import assert_equal, assert_is_not, assert_is_true, assert_is_not_true
+from step_helpers import assert_equal, assert_is_not, assert_is_true, assert_is_not_true
```

Comment:
oh this is great.  I wasn't sure wtf was going on with the import statements but this makes it INFINITELY easier to read.  Now just have to figure out why PyCharm doesn't recognize this as a valid import

Reply:
(no action needed here btw)

Reply:
You might have to mark the test directory as root. You can do that by right clicking on it and going to “mark directory as”.